src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.cs

```diff
@@ -321,13 +322,13 @@ private List<DeploymentOperation> GetNewOperations(List<DeploymentOperation> old
             return newOperations;
         }
 
-        public Management.Resources.Models.ProviderOperationsMetadata GetProviderOperationsMetadata(string providerNamespace)
+        public RMProviderOperationsMetadata GetProviderOperationsMetadata(string providerNamespace)
         {
             ProviderOperationsMetadataGetResult result = this.ResourceManagementClient.ProviderOperationsMetadata.Get(providerNamespace);
             return result.Provider;
         }
 
-        public IList<Management.Resources.Models.ProviderOperationsMetadata> ListProviderOperationsMetadata()
+        public IList<RMProviderOperationsMetadata> ListProviderOperationsMetadata()
```

Comment:
could you run all the tests once in record mode and include the new session records,Just want to make sure the move to package 2,5,0 wont break any of the tests